plugins/janus_videoroom.c

```diff
@@ -2321,6 +2330,43 @@ void janus_videoroom_hangup_media(janus_plugin_session *handle) {
 	}
 }
 
+static void janus_videoroom_sdp_v_format_vp8(char *mline, int mline_size, int pt, int b, const char *video_mode) {
```

Comment:
A way to be able to use the same `*printf()` call, despite there being a different number of substitutions, is to specify the argument position in the format string. For every instance of `pt` after the first, the format string could have `%1$d` which means "take the first value argument as type d" (instead of "take the next argument"), and you would not need to provide the subsequent `pt` parameters to `g_snprintf()`